core/src/Cuda/Kokkos_Cuda_Graph_Impl.hpp

```diff
@@ -200,6 +200,9 @@ struct GraphImpl<Kokkos::Cuda> {
         m_execution_space, _graph_node_kernel_ctor_tag{},
         aggregate_kernel_impl_t{});
   }
+
+  const cudaGraph_t     getNativeGraph()           const { return m_graph; }
```

Comment:
@dalg24 I don't know what's best. Make `m_graph` public or use a getter?

I don't have a strong opinion on any.

Reply:
A getter for sure.
Why is your return type const-qualified?

Reply:
Sorry, `const` qualification makes no sense here. Removed.